src/app/functions/server/deleteAccount.ts

```diff
@@ -37,6 +39,21 @@ export async function deleteAccount(
     logger.info("deactivated_onerep_profile", {
       subscriber_id: subscriber.id,
     });
+
+    // try to unsubscribe from subplat
+    try {
+      const subscriptionId = "monitor";
+      await deleteSubscription(subscriptionId);
+      logger.info("unsubscribe_from_subplat", {
+        subscriber_id: subscriber.id,
+        success: true,
+      });
+    } catch (ex) {
+      logger.error("unsubscribe_from_subplat", {
```

Comment:
Should L47 and L52 use the same `unsubscribe_from_subplat` key/string? Or should this be something like `unsubscribe_from_subplat_failed`?

Reply:
I think the `logger.error` would show up differently enough in our logs that this won't be misinterpretted